packages/fetchai/connections/http_server/connection.py

```diff
@@ -460,6 +461,10 @@ async def _http_handler(self, http_request: BaseRequest) -> Response:
 
         except asyncio.TimeoutError:
             return Response(status=REQUEST_TIMEOUT, reason="Request Timeout")
+        except FuturesCancelledError:
```

Comment:
If there is an error during serving a request, then the AEA tears down. We need to then mute this error, otherwise we cannot see the original exception.